### PR TITLE
Add new answers to "randomanswer"

### PR DESCRIPTION
**Pull request by Bryan-0**
- Add __3__ new answers to ``.randomanswer``.

### CODE REVIEW

File: commands/misc.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ exports.commands = {
 	randomanswer: function (arg, user, room) {
 		if (room === user) return false;
 		var text = '';
-		var rand = ~~(20 * Math.random());
+		var rand = ~~(23 * Math.random());
```
Comment (Freigeist) [inline commands/misc.js]: Only updating the spanish language could and will cause errors, should be dynamic anyway (`answers.length`)

Comment (Bryan-0) [inline commands/misc.js]: Oh, so, I need to update all languages, right? with the __3__ random answers.

Comment (Freigeist) [inline commands/misc.js]: More like:
```js
var text = '';
var answers = this.trad('answers');
var rand = ~~(answers.length * Math.random());
```

With this solution, it should be no problem, if different languages have different amounts of answers.

Tagging @Ecuacion for a opinion on this case.

Comment (Bryan-0) [inline commands/misc.js]: Alright.

Comment (Ecuacion) [inline commands/misc.js]: Problem fixed in commit 57638e9

**Comment by Bryan-0** [comment]
I just copy again your commit 57638e9 @Ecuacion, becuase there was a "Merge Conflicts" if you don't want this, I can close the pr and open another.

**Pull request merged**